imap_processing/cli.py

```diff
@@ -356,7 +357,8 @@ def pre_processing(self):
         return self.download_dependencies()
 
     @abstractmethod
-    def do_processing(self, dependencies: list):
+    def do_processing(self, dependencies: list):  # type: ignore[no-untyped-def]
+        # Todo Remove type: ignore once function is implemented.
```

Comment:
```suggestion
    def do_processing(self, dependencies: list):  # type: ignore[no-untyped-def]
```

Since this is an abstract method that is meant to be implemented in child classes, I don't think it will ever actually be 'implemented' here.

So I think it is fine to keep the inline `ignore` statement here.

Reply:
Oh yeah this is what I meant, agree with Matt here